canonical-websites/snapcraft.io
# Filter revisions table based on arch clicked

Fixes #1225 

Adds clickable 'unassigned revisions' to release table, where users can choose to select revisions from popover with revisions filtered by architecture.

### QA

- ./run or https://snapcraft-io-canonical-websites-pr-1257.run.demo.haus/
- go to Releases page of any snap
- 'Unassigned revisions' row show be visible in releases table
- Click on any '+ Add revision'
- Popover should show up with latest revisions from given architecture
- You can select a revision from the list, it will show in the releases table, ready to be promoted to some channel

<img width="1036" alt="screen shot 2018-11-02 at 08 07 25" src="https://user-images.githubusercontent.com/83575/47898530-8f827980-de76-11e8-94b2-29e043dca366.png">
